### PR TITLE
Add task priorities and possibility to remove

### PR DESCRIPTION
**Pull request by dgrochowski**

### CODE REVIEW

File: taskmanager/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -132,4 +132,37 @@ public function convertUserNameToUser(string $user): User
 
         return reset($userCollection);
     }
+
+    /**
+     * @When I change task named :task priority to :priority
+     */
+    public function iChangeTaskNamedPriorityTo(Task $task, string $priority)
```
Comment (tarnawski) [inline taskmanager/features/bootstrap/FeatureContext.php:139]: To be consistent maybe better to do priority as object like statuses

File: README.md [hunk 2]
```diff
@@ -5,8 +5,19 @@ Self learning project
 ## Installation
 Clone repository
 
+## How to launch
+
 ```bash
 docker-compose up
+docker exec -it tm_php composer install
```
Comment (tarnawski) [inline README.md:12]: Nice to have this composer instruction. IMHO good tip is run composer locally then composer get packages from cache (of course if you use they before in this or other project).

**Comment by dominikhajduk** [comment]
@dgrochowski please open PR from your fork https://github.com/PGS-php/upskilling-2nd-edition/blob/develop/CONTRIBUTING.md

**Comment by dgrochowski** [comment]
Closed because of: https://github.com/PGS-php/upskilling-2nd-edition/pull/21

**Pull request closed without merging**